obfx_modules/mystock-import/js/src/components/index.js

```diff
@@ -0,0 +1,20 @@
+import PhotoList from "./PhotoList";
+import SetFeaturedImage from "./SetFeaturedImage";
+import InsertImage from "./InsertImage";
+
+const { PluginSidebar, PluginSidebarMoreMenuItem } = wp.editPost;
```

Comment:
You have imported `PluginSidebarMoreMenuItem` but not used it. It's not working on my without adding `PluginSidebarMoreMenuItem`